.github/workflows/ci.yml

```diff
@@ -1,92 +1,96 @@
-name: CI
-
-on:
-  pull_request:
-    types: [opened, synchronize]
-  push:
-    branches:
-      - develop
-      - stable
-      - ci/*
-
-jobs:
-  build:
-    runs-on: ubuntu-latest
-    steps:
-      - uses: actions/checkout@v2
-
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 14.x
-
-      - uses: bahmutov/npm-install@v1
-        with:
-          # By default, bahmutov/npm-install uses --frozen-lockfile
-          install-command: yarn
-
-      - name: Run tests and collect coverage
-        run: yarn test --coverage
-
-      - name: Typecheck
-        run: yarn typecheck
-
-      - name: Build
-        run: yarn build
-
-      - name: Run Codechecks
-        run: yarn codechecks
-        env:
-          CC_SECRET: ${{ secrets.CC_SECRET }}
-        if: ${{ env.CC_SECRET != '' }}
-
-  # Dependabot and PRs from forks should not release canaries,
-  # but secrets and env vars cannot be read in `job.if`, so we check if
-  # GH_TOKEN is present before attempting to release.
-  release_check:
-    runs-on: ubuntu-latest
-    needs: [build]
-    steps:
-      - id: can_release
-        run: echo '::set-output name=CAN_RELEASE::${{ env.CAN_RELEASE }}'
-    env:
-      CAN_RELEASE: ${{ secrets.AUTO_RELEASE_GH_TOKEN != '' }}
-    outputs:
-      CAN_RELEASE: ${{ steps.can_release.outputs.CAN_RELEASE }}
-
-  release:
-    runs-on: ubuntu-latest
-    needs: [release_check]
-    if: >
-      !contains(github.event.head_commit.message, 'ci skip') &&
-      !contains(github.event.head_commit.message, 'skip ci') && (
-        github.event_name == 'push' ||
-        github.event.pull_request.head.repo.full_name == github.repository
-      ) && (
-        needs.release_check.outputs.CAN_RELEASE == 'true'
-      )
-    steps:
-      - uses: actions/checkout@v2
-
-      - name: Prepare repository
-        run: git fetch --unshallow --tags
-
-      - uses: actions/setup-node@v1
-        with:
-          node-version: 14.x
-
-      - uses: bahmutov/npm-install@v1
-
-      - name: Queue in release turnstile
-        id: turnstyle
-        continue-on-error: true
-        uses: softprops/turnstyle@v1
-        with:
-          abort-after-seconds: 600
-
-      - name: Create release
-        if: steps.turnstyle.outcome == 'success'
-        env:
-          GITHUB_TOKEN: ${{ secrets.AUTO_RELEASE_GH_TOKEN }}
-          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
-          NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
-        run: yarn release
+name: CI
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+  push:
+    branches:
+      - develop
+      - stable
+      # - ci/* # uncomment this when workin on CI without PR
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1

```

Comment:
we could consider v2 for that one perhaps?
https://github.com/actions/setup-node/releases/tag/v2.2.0